app/server/src/main/scala/org/bitcoins/server/BitcoinSServerMain.scala

```diff
@@ -147,15 +148,14 @@ class BitcoinSServerMain(override val args: Array[String])
 
         blockCount <- bitcoind.getBlockCount
         // Create callbacks for processing new blocks
-        _ = bitcoindRpcConf.zmqPortOpt match {
-          case Some(_) =>
-            val zmq = BitcoindRpcBackendUtil.createZMQWalletCallbacks(wallet)
-            zmq.start()
-          case None =>
+        _ =
+          if (bitcoindRpcConf.zmqConfig == ZmqConfig()) {
```

Comment:
nit: I would make a `ZmqConfig.empty` as that is more clear as to what is going on here IMO